source/application/SXRMB/SXRMB.h

```diff
@@ -18,6 +18,10 @@
 
 namespace SXRMB {
 
+	enum SXRMBErrorCodes {
+		ERR_SXRMB_BEAMLINE_SHUTTERS_TIMEOUT = 290101  // 2-CLS 9-SXRMB 01 module-01 errorID-01
```

Comment:
Interesting twist on our normal way of doing this @XuetaoLiu-CLS 
If you don't want it to be a #define and use an enum, then I would suggest using camel case.
Also, no need to skimp on the extra letters ... I would go for "ErrorSXRMB..."